client/src/components/SaveBudget.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+const SaveBudget = (sum) => {
+  const [value, setValue] = useState(sum.budget);
+
+  const handleChange = (event) => {
+    setValue(event.target.value);
+  };
+
+  const handleClick = () => {
+    sum.handleEditClick(value);
+  };
+
+  return (
+    <>
+      <Input
+        placeholder="Click on Edit..."
+        type="number"
+        id="save"
+        value={value}
+        disabled
+        onChange={handleChange}
+      />{" "}
+      ,- €
+      <div>
+        <button onClick={handleClick}>Edit</button>
+      </div>
+    </>
+  );
+};
+
```

Comment:
This component again looks quite much the same as `EditBudget` and `EditTrip` and could become one with the those? ;)